services/src/integ-test/java/uk/gov/moj/sdt/services/utils/ExecuteScriptServiceTest.java

```diff
@@ -100,21 +109,19 @@ void testExecuteScript() {
     @Test
     void testRunScript() {
         // check the records do not exist yet
-        IBulkSubmission bulkSubmission = bulkSubmissionDao.getBulkSubmissionBySdtRef(bulkCustomer, sdtBulkReference,
-                dataRetentionPeriod);
-        assertNull(bulkSubmission);
-
+        assertThrows(NoResultException.class, () -> bulkSubmissionDao.getBulkSubmissionBySdtRef(bulkCustomer, sdtBulkReference,
+                                                                                               dataRetentionPeriod));
```

Comment:
Indentation of dataRetentionPeriod parameter is slightly out.

Reply:
corrected it